sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRDD.scala

```diff
@@ -68,7 +73,9 @@ object JDBCRDD extends Logging {
         statement.close()
       }
     } finally {
-      conn.close()
+      if (null != conn) {
```

Comment:
It can't be null here right?

Reply:
yes, it can't be null now because we throw exception before.
I just forgot to remove it